contents/docs/_snippets/identify-setting-user-properties.mdx

```diff
@@ -1,3 +1,3 @@
-You'll notice that one of the parameters in the `identify` method is a `properties` object. This enables you to set [user properties](/docs/getting-started/user-properties). Whenever possible, we recommend passing in all user properties you have available each time you call identify, as this ensures their user profile on PostHog is up to date.
+You'll notice that one of the parameters in the `identify` method is a `properties` object. If you are capturing person profiles, this enables you to set [user properties](/docs/getting-started/user-properties). Whenever possible, we recommend passing in all user properties you have available each time you call identify, as this ensures their person profile on PostHog is up to date.
```

Comment:
```suggestion
You'll notice that one of the parameters in the `identify` method is a `properties` object. This enables you to set [user properties](/docs/getting-started/user-properties). Whenever possible, we recommend passing in all user properties you have available each time you call identify, as this ensures their person profile on PostHog is up to date.
```

because they are calling `identify`, they will automatically have a person profile